app/controllers/vms_controller.rb

```diff
@@ -46,6 +46,8 @@ def edit
     @sudo_user_ids = @vm.sudo_users.map(&:id)
     @non_sudo_user_ids = @vm.users.map(&:id)
     @description = @vm.ensure_config.description
+    @ip = @vm.ensure_config.ip
```

Comment:
please use a single @configuration @vm_config or similar instance variable, instead of creating more local variables